assets/js/translate-analytics.js

```diff
@@ -14,6 +14,10 @@ export default () => {
   if (typeof MutationObserver === "function") {
     // Tell the observer to monitor for changes to HTML attributes
     const config = { attributes: true };
+
+    // Get the language of the page on load.
+    const oldCookieLanguage = getCookieLanguage();
```

Comment:
The rest of this file's code identifies the language by checking the `lang=` attribute's value on the `<html>` tag. Both approaches appear to work great, I'm just curious whether we need this additional approach as opposed to you doing exactly what you do here but using `document.querySelector("html").getAttribute("lang")` to get the language instead of parsing the cookie.

Reply:
We **have** to check the cookie to get the language change because it won't be in the HTML yet. This just makes it so that I'm checking the same source before and after the change. I could check the `lang` and then check the cookie. But, that might be more confusing.